src/alembic/env.py

```diff
@@ -58,6 +59,7 @@ async def run_async_migrations() -> None:
     """
     configuration = config.get_section(config.config_ini_section)
     configuration["sqlalchemy.url"] = database.SQLALCHEMY_DATABASE_URL
+    # configuration["sqlalchemy.url"] = database.SQLALCHEMY_TEST_DATABASE_URL
```

Comment:
the purpose of the test db is to run tests from scratch (without migrations), so we should remove this comment as we don't want to accidentally do so